src/style.css

```
@@ -1 +1,87 @@
-/* add styles here */
+*{
+  box-sizing: border-box;
+}
+
+@font-face {
+  font-family: Avenir;
+  src: url(./fonts/Avenir-Book.ttf);
+}
+
+html {
+  font-family: Avenir, sans-serif ;
+  font-weight: 300;
+}
+
+/* body {
+  display: flex;
+  flex-direction: column;
+  align-items: center;
+  justify-content: center;
+  background-color: #f0f0f0;
+} */
+
+.search-bar {
+  display: flex;
+  justify-content: center;
+
+  margin-top: 20px;
+  position: relative;
+}
+
+.search-input {
+  flex: 1;
+  border: 1px solid #E1E7ED;
+  border-radius: 4px;
+  background-image: url(images/Search.svg);
+  background-repeat: no-repeat;
+  background-color: #FFF;
+  font-weight: 300;
+  color: #3D4E61;
+  outline: none;
+  width: 100%;
+  font-family: inherit;
+}
+
+
+.search-input--big {
+  font-size: 16px;
+  height: 70px; /*  Big search bar line height */
```

Remove comments everywhere
```suggestion
  height: 70px;
```